practice-one/src/assets/css/footer.css

```diff
@@ -0,0 +1,60 @@
+.footer-section {
+  background-color: var(--black-color);
+  padding-top: 105px;
+  padding-left: 147px;
+  padding-right: 151px;
+  padding-bottom: 91px;
+}
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-item-top {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+.footer-item-bottom {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.item-top {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.contact-us {
+  display: flex;
+  flex-direction: row;
+  gap: 54px;
+}
+.follow-us {
+  display: flex;
+  gap: 43px;
+}
+.break {
+  max-width: 133px;
+  max-height: 77px;
+}
+.break-about-us {
+  max-width: 360px;
+  max-height: 48px;
+}
+.rule {
+  width: 100%;
+  height: 1px;
+  background-color: var(--white-color);
+  border-style: solid;
+  margin-top: 42px;
+  margin-bottom: 38px;
+}
+.opacity {
```

Comment:
Why do we need this class?

Reply:
Because the Footer section has a white horizontal line, and I am using this class to style that horizontal line. As for opacity class, I am using it to apply opacity to the text and icons in the Footer section.

Reply:
Is it only used in the Footer section or any place else? Another comment is the value of that property is too long. You should be rounding numbers.

Reply:
Yes, it is only used in the Footer section

I fixed it
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/d7381a06-c65f-430a-bed1-badd096ae6a8)